pkg/minikube/cluster/cluster.go

```diff
@@ -192,6 +194,8 @@ type MachineConfig struct {
 func engineOptions(config MachineConfig) *engine.Options {
 	o := engine.Options{
 		Env:              config.DockerEnv,
+		AddRegistry:      config.AddRegistry,
+		BlockRegistry:    config.BlockRegistry,
```

Comment:
engine Options struct doesn't have those variable which you are trying to set value for. Check options which are currently present for Engine https://github.com/docker/machine/blob/master/libmachine/engine/engine.go